c2corg_ui/static/partials/cards/images.html

```diff
@@ -0,0 +1,16 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <img ng-src="{{cardCtrl.createImg()}}" alt="{{locale['title']}}">
```

Comment:
I'd love to show you but the images loaded only once and i cannot find them anymore :P SI are distorted because they are simply too small, you can see, even on the cards, that they are pixelated (and they are squared as well).